yarn.lock

```diff
@@ -3825,8 +3872,8 @@ __metadata:
   linkType: hard
 
 "@metamask/snaps-controllers@npm:^9.10.0":
-  version: 9.13.0
-  resolution: "@metamask/snaps-controllers@npm:9.13.0"
+  version: 9.12.0
```

Comment:
Seems like a rebase gone wrong potentially?

Reply:
i've tried yarn and yarn dedupe and removing my node_modules. My yarn lock doesn't change

Reply:
deleting my yarn.lock and running yarn and deduping again gives me this https://github.com/MetaMask/core/pull/4813/commits/20588efc71c848bba74884cc8a514f6cfcbc8f5b

Reply:
this package resolves to 9.17.0 on this line now

Reply:
I wouldn't delete yarn.lock as that re-resolves many dependencies. Can you try reverting to the yarn.lock on main and running yarn? That should help!

Reply:
having a hard time understanding how the diff represented [here](https://github.com/MetaMask/core/pull/4813/commits/8fff3517262e111dc2702e0b5f5acfa42b153cde) is netting out to only 123 line additions in the [final diff for the PR](https://github.com/MetaMask/core/pull/4813/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de) 🤷

Reply:
The diff was huge before due to Jiexi deleting his `yarn.lock`, this huge diff undoes the old one 😄